core-bundle/src/Resources/contao/classes/Backend.php

```diff
@@ -336,8 +336,8 @@ protected function getBackendModule($module, PickerInterface $picker = null)
 		/** @var Session $objSession */
 		$objSession = System::getContainer()->get('session');
 
-		$arrTables = (array) $arrModule['tables'];
-		$strTable = Input::get('table') ?: $arrTables[0];
+		$arrTables = $arrModule['tables'] ?? array();
```

Comment:
```suggestion
		$arrTables = (array) ($arrModule['tables'] ?? array());
```

I think this array cast has to stay. If somone sets `'tables' => 'tl_my_table'` this would now not be converted to `array('tl_my_table')` anymore.